src/common/scripts/services.js

```diff
@@ -0,0 +1,31 @@
+(function (angular) {
+    'use strict';
+
+    var module = angular.module('user_management.common');
+
+    module.service('catchErrors', [
+        function () {
+            this.all = function (response, fields) {
```

Comment:
This is very similar to https://github.com/incuna/incuna-epatient-web/blob/master/epatient/modules/epatient/scripts/epatient.js#L6 

The long term goal should be to move the functionality to a new library / module that is used by both, if this is not done now it would make sense to make this the same as `fieldErrors`.